### PR TITLE
Improvements in Login Screen UI

### PR DESCRIPTION
**Pull request by Hiteshgautam01**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the checks with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

UI after Changes 

whatsapp image 2018-03-10 at 12 40 42 am

### CODE REVIEW

**Comment by saketkumar95** [comment]
Also don't change the design of `Create a myExperiment account`

**Comment by sagar15795** [comment]
@Hiteshgautam01 Either use 9-patch image or XML drawable for the background. Also, remove the background of `Create a myExperiment account`.

**Comment by Hiteshgautam01** [comment]
Hey @sagar15795 @saketkumar95 
Why we are removing the background of create a myExperiment account. ?

**Comment by Hiteshgautam01** [comment]
UI after changes

updated login

File: app/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -32,8 +32,8 @@
 
         <ImageView
             android:id="@+id/logo"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            android:layout_width="70dp"
+            android:layout_height="70dp"
```
Comment (saketkumar95) [inline app/src/main/res/layout/fragment_login.xml]: Adds these all in `dimens.xml` file

File: app/src/main/res/drawable/login_button.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle" android:padding="10dp">
+    <!-- you can use any color you want I used here gray color-->
```
Comment (sagar15795) [inline app/src/main/res/drawable/login_button.xml]: remove this unwanted comment

File: app/src/main/res/drawable/login_button.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    <solid android:color="@color/colorPrimaryDark"/>
+    <corners android:radius="80dp"/>
```
Comment (sagar15795) [inline app/src/main/res/drawable/login_button.xml]: Move this to `dimens.xml`

**Comment by Hiteshgautam01** [comment]
Hey @sagar15795 why we are removing the `create a myExperiment account` background. It does not look even to the login screen I think.

**Comment by sagar15795** [comment]
@Hiteshgautam01 because the main action on this screen is login.

**Pull request merged**